src/components/Roadmap.tsx

```diff
@@ -0,0 +1,56 @@
+import classNames from 'classnames';
+import data from 'data.json';
+
+//styles
+import '../index.scss';
+
+//types
+import { Status, STATUSES } from 'types';
+
+export const Roadmap = () => {
+  let plannedCount = data.productRequests.filter((item) => {
+    return item.status === 'planned';
+  }).length;
+
+  let inProgressCount = data.productRequests.filter((item) => {
+    return item.status === 'in-progress';
+  }).length;
+
+  let liveCount = data.productRequests.filter((item) => {
+    return item.status === 'in-progress';
+  }).length;
+
+  let statuses: Status[] = [
+    { status: STATUSES.PLANNED, count: plannedCount },
+    { status: STATUSES.IN_PROGRESS, count: inProgressCount },
+    { status: STATUSES.LIVE, count: liveCount },
+  ];
+
+  return (
+    <section className="roadmap">
+      <div className="roadmap__text">
+        <span>Roadmap</span>
+        <span>View</span>
+      </div>
+      <div className="roadmap__list">
+        {statuses.map((status: Status) => {
+          const dotClass = classNames({
```

Comment:
Suggestion for cleanup - destructure!

```
{statuses.map((s: Status) => {
  const {status, count} = s;
  ...
}
```

Reply:
Ok, would like to work through that on our next meet up.

Reply:
Sure thing